### PR TITLE
Add docs section

### PR DESCRIPTION
**Pull request by NataliiaKononikhina**
Screenshot 2019-03-26 at 10 24 39

### CODE REVIEW

File: source/theme/styles/components/landing-page-docs-section.scss [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+  }
+}
+
+@media (max-width: $mobile-navigation-breakpoint) {
+  .docs {
+    padding: 80px 37px 110px;
+  }
+
+  .docs-section {
+    overflow-x: auto;
```
Comment (kparzecki) [inline source/theme/styles/components/landing-page-docs-section.scss]: `overflow-x: auto` should be applied on `.docs`

**Comment by kparzecki** [comment]
Screenshot 2019-03-26 at 15 10 44

I think it would be better to align `DIG INTO THE DOCS` header with page content.

File: source/theme/landing-page-docs-section.html [hunk 2]
```diff
@@ -0,0 +1,22 @@
+<div class="section docs">
+  <h3 class="name-section">Dig into the docs</h3>
+  <div class="section-wrapper docs-section">
+    <article class="docs__article">
+      <img src="./_static/img/guides.png" class="article__graphics">
```
Comment (kanstantsin-nekhniadovich) [inline source/theme/landing-page-docs-section.html]: Could you please add alt attribute to img.

File: source/theme/landing-page-docs-section.html [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+      <h4 class="article__title">Guides</h4>
+      <p class="article__text">Hit the ground running with our step by step guides for developers. Built by developers, for developers.</p>
+    </article>
+    <article class="docs__article">
+      <img src="./_static/img/store-api.png" class="article__graphics">
```
Comment (kanstantsin-nekhniadovich) [inline source/theme/landing-page-docs-section.html]: And here.

File: source/theme/landing-page-docs-section.html [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+      <h4 class="article__title">Store API</h4>
+      <p class="article__text">Hit the ground running with our step by step guides for developers. Built by developers, for developers lorem ipsum lorem ipsum and more and more.</p>
+    </article>
+    <article class="docs__article">
+      <img src="./_static/img/order-api.png" class="article__graphics">
```
Comment (kanstantsin-nekhniadovich) [inline source/theme/landing-page-docs-section.html]: and here.

File: source/theme/landing-page-docs-section.html [hunk 5]
```diff
@@ -0,0 +1,24 @@
+<div class="section docs">
```
Comment (paszkowskiDamian) [inline source/theme/landing-page-docs-section.html]: use `section`

File: source/theme/landing-page-hero-section.html [hunk 6]
```diff
@@ -1,5 +1,5 @@
-<div class="hero-section">
-  <div class="hero-section__wrapper">
+<div class="hero-section section">
```
Comment (paszkowskiDamian) [inline source/theme/landing-page-hero-section.html]: <--

**Pull request merged**